packages/starksheet-webapp/src/contracts/worksheet/EvmWorksheetContract.ts

```diff
@@ -0,0 +1,72 @@
+import BN from "bn.js";
+import { Contract } from "ethers";
+import { JsonRpcProvider } from "@ethersproject/providers";
+import { ABI, CellData, CellRendered, WorksheetContract } from "../../types";
+
+export class EvmWorksheetContract implements WorksheetContract {
+  private contract: Contract;
+
+  /**
+   * The class constructor.
+   */
+  constructor(private address: string, private abi: ABI, private provider: JsonRpcProvider) {
+    this.contract = new Contract(address, abi, provider);
+  }
+
+  /**
+   * @inheritDoc
+   */
+  async getCell(tokenId: number): Promise<CellData> {
+    const [contractAddress, value, data] = await this.contract.getCell(tokenId);
+    return {
+      contractAddress,
+      selector: value,
+      calldata: data,
+    };
+  }
+
+  /**
+   * @inheritDoc
+   */
+  getCellPrice(): Promise<number> {
+    throw new Error("not implemented");
+  }
+
+  /**
+   * @inheritDoc
+   */
+  nRow(): Promise<number> {
+    throw new Error("not implemented");
```

Comment:
same as above, keep the hard coded default constants of the StarknetWorksheet contract plz